### DISCUSSION TITLE
problems with using drawString() to Displays a variable-length string

### DISCUSSION THREAD

**Discussion by baloyou** [Bodmer/TFT_eSPI#1997] [Q&A - General] [answered]
hello, bodmer

i have a problems about `drawString()`

it's normal. 

```
tft.drawString("12345",0,0);
```

but ,when i display "1234",The last 5 entered did not disappear

```
tft.drawString("1234",0,0);
```

I can only type this way(add a space after 4)

```
tft.drawString("1234 ",0,0);
```

Is there a better way?

thanks!!

**Comment by Bodmer** [comment, marked as answer]
Use the tft.setTextPadding(w) to define the width in pixels of the area to be blanked.
